homeassistant/components/moehlenhoff_alpha2/button.py

```diff
@@ -0,0 +1,42 @@
+"""Button entity to set the time of the Alpha2 base."""
+
+from datetime import datetime
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import Alpha2BaseCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Add Alpha2 button entities."""
+
+    coordinator: Alpha2BaseCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    async_add_entities([Alpha2TimeSyncButton(coordinator, config_entry.entry_id)])
+
+
+class Alpha2TimeSyncButton(CoordinatorEntity[Alpha2BaseCoordinator], ButtonEntity):
+    """Alpha2 virtual time sync button."""
+
+    _attr_name = "Sync Time"
```

Comment:
```suggestion
    _attr_name = "Sync time"
```

https://developers.home-assistant.io/docs/core/entity#entity-naming

Side note: This integration doesn't seem to implement device info. That could be a future improvement.

https://developers.home-assistant.io/docs/device_registry_index